### PR TITLE
Make cl workers show user-owned workers for non-root users

### PR DESCRIPTION
**Pull request by nelson-liu**
I think the cl workers output is _super_ helpful, but it's locked behind root restrictions. This PR makes the command output the individual user's workers, if the user is not root.

I couldn't find a test for this anywhere...maybe I'm missing something?

### CODE REVIEW

File: codalab/lib/bundle_cli.py [hunk 1]
```diff
@@ -1034,7 +1034,10 @@ def auto_convert_type(value):
 
     @Commands.command(
         'workers',
-        help=['Display worker information of this CodaLab instance. Root user only.'],
+        help=[
+            'Display worker information of this CodaLab instance. '
+            'If not root user, only display information about your connected workers.'
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Change: "Display information about workers that you have connected to the CodaLab instance." (most people won't need to worry about the root user).

File: codalab/rest/workers.py [hunk 2]
```diff
@@ -135,10 +135,10 @@ def start_bundle(worker_id, uuid):
 
 @get("/workers/info", name="workers_info", apply=AuthenticatedPlugin())
 def workers_info():
-    if request.user.user_id != local.model.root_user_id:
-        abort(http.client.UNAUTHORIZED, "User is not root user")
-
     data = local.worker_model.get_workers()
```
Comment (percyliang) [inline codalab/rest/workers.py]: rename `data` to `workers`

File: docs/CLI-Reference.md [hunk 3]
```diff
@@ -449,7 +449,7 @@ Usage: `cl <command> <arguments>`
 
 ## Commands for managing server:
   ### workers:
-    Display worker information of this CodaLab instance. Root user only.
+    Display worker information of this CodaLab instance. If not root user, only display information about your connected workers.
```
Comment (percyliang) [inline docs/CLI-Reference.md]: Use the description from above, but add: "For the root user, display all workers."

**Review by percyliang — CHANGES_REQUESTED** [review]
Thanks, this is a good idea!  Just a few small cosmetic suggestions.

**Review by percyliang — APPROVED** [review]

File: docs/CLI-Reference.md (hunk 3 above)
Comment (nelson-liu) [reply docs/CLI-Reference.md]: Note that the pre-commit script will overwrite this change every time to match the output of `cl -h -m` (i.e., removing `For the root user, display all workers.`).

**Pull request merged**